src/components/Rapidoc.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Helmet from 'react-helmet'
+
+const Rapidoc = () => {
+  return (
+    <div className='container docs-page'>
+      <Helmet>
+        <script type="module" src="https://unpkg.com/rapidoc/dist/rapidoc-min.js"></script>
+        <style type="text/css">{`
+        .main-content-inner--view-mode{
```

Comment:
I am unable to write a working selector for this class. I think it is `.jsx` kebab-case weirdness.

Reply:
Going to ignore this. If anybody knows how to gain the additional 32px, I’d love to push a change.